packages/selected-network-controller/CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- **BREAKING**: Bump dependency on `@metamask/network-controller` to ^13.0.0 ([#1633](https://github.com/MetaMask/core/pull/1633))
```

Comment:
Nit: 
```suggestion
- **BREAKING**: Bump peer dependency on `@metamask/network-controller` to ^13.0.0 ([#1633](https://github.com/MetaMask/core/pull/1633))
```

Reply:
Done in https://github.com/MetaMask/core/pull/1708/commits/c96a938ce8988f6037fbf7cc3ffbdf41401438e5